nodejs/src/calculator/index.js

```diff
@@ -0,0 +1,59 @@
+import 'dotenv/config';
+require('es6-promise').polyfill();
+
+import SnetSDK, { DefaultPaymentChannelManagementStrategy } from 'snet-sdk';
+
+import services from './example_service_grpc_pb';
+import messages from './example_service_pb';
+import config from '../config';
+
+const sdk = new SnetSDK(config);
+
+const main = async () => {
+  const orgId = 'snet';
+  const serviceId = 'example-service';
+  const groupName = 'default_group';
+  let defaultPaymentChannelManagementStrategy = new DefaultPaymentChannelManagementStrategy(sdk);
+
+  const serviceClient = await sdk.createServiceClient(orgId, serviceId, groupName, defaultPaymentChannelManagementStrategy, services.CalculatorClient);
```

Comment:
1. Is it really necessary to pass `service.CalculatorClient` as last parameter? We could export all services as part of `stub` callection of the `serviceClient`.
2. As far as I remember @vforvalerio87 suggested using `service` instead of `stub` and adding `serviceClient.messages` to return messages classes

Reply:
Resolved in [PR](https://github.com/singnet/snet-sdk-js/pull/1).